### ISSUE TITLE
PHPStorm autocomplete doesn't work

### ISSUE THREAD

**Issue by Jurigag** [Codeception/Verify#22]
I added Verify,Specify and Codeception as my External Libraries to my project so i have auto complete in my tests.

when doing for example:

`except($something)->`

PHPStorm don't provide methods which are in `Verify` class. Same is happening with verify.

**Comment by Insolita** [comment]
+1

**Comment by Jurigag** [comment]
Also i dont get it. Sometimes it works, sometimes it doesn't.

**Comment by jakim** [comment]
/** @return Verify */
verify($something)->

**Comment by kt81** [comment]
I've got the same problem. using Intellij 2016.1 with PHP plugin.
It seems like the PhpStorm's bug?

```php
class Test extends PHPUnit_Framework_TestCase
{
	/**
	 * @return \Codeception\Verify
	 */
	protected function verify()
	{
		return verify(func_get_args());
	}
	
	public function testMe()
	{
		$a = 1;
		
		// auto complition does not works
		verify('test', $a)->equals(1);

		// this works. i don't know why it works only such this case.
		verify($a)->equals(1);
		
		// when same function declared inside some class, it works.
		$this->verify('test', $a)->equals(1);
	}
}
```

**Comment by Jurigag** [comment]
It is supposed to work on 162.917 phpstorm build. Can someone check it ?
https://youtrack.jetbrains.com/issue/WI-32230#comment=27-1481399

**Comment by kt81** [comment]
@Jurigag 
First, I don't know it could help your case, I figured out a very simple solution.
Add ```\``` before ```verify()``` and ```expect()```, or delete testcase's namespace, then autocompletion works fine.

Anyway, the problem has still occurred in v162.917.12.
I found that it is more complicated problem.

I made a new clean Laravel project with verify and created a simple test class.
```php
<?php
namespace test;

class Test extends \PHPUnit_Framework_TestCase
{
    protected function testA()
    {
        $a = 1;
        verify('＼( ‘ω’)／ウオオアアアアアアアアアーッッッッッッ！！', $a)->equals(1);
    }
}
```
In this case, autocomplete has worked correctly.
Then I simply coppied this code to my project that has the autocomplete problem.
However, autocomplete has not worked. (and a inspection warning occurred as usual...)
I think this problem is related by other libraries but I couldn't figure out the causes.
